### ISSUE TITLE
Crash in Debian 12

### ISSUE THREAD

**Issue by dkuteynikov** [micheleg/dash-to-dock#2050]
Dash to Dock 84
Debian GNU/Linux 12 (bookworm)
Gnome 43.4
Wayland

```
Object St.Label (0x559c93477c50), has been already disposed — impossible to access it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs.
== Stack trace for context 0x559c8f8fb7b0 ==
#0   7fff6207f5f0 b   resource:///org/gnome/shell/ui/dash.js:85 (212da21c4ba0 @ 65)
#1   559c92d089c8 i   /home/dmitry/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/docking.js:475 (26f8f3d2650 @ 22)
#2   7fff62081830 b   self-hosted:1121 (fc6cf07eec0 @ 432)
```
See details in the attached files.
[xsession-errors.log](https://github.com/micheleg/dash-to-dock/files/11839649/xsession-errors.log)
[gnome-shell.log](https://github.com/micheleg/dash-to-dock/files/11839656/gnome-shell.log)
[journalctl_xe.log](https://github.com/micheleg/dash-to-dock/files/11839659/journalctl_xe.log)
[dconf.dump.txt](https://github.com/micheleg/dash-to-dock/files/11839663/dconf.dump.txt)

**Comment by 3v1n0** [comment]
It seems to be a problem inside the upstream dash code, however that shouldn't cause any crash.

**Comment by vanvugt** [comment]
I think dash-to-dock is just a victim since this keeps happening:
```
июн 16 22:19:17 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:17 dk-pavilion-laptop gnome-shell[5247]: meta_later_remove: assertion 'display' failed
июн 16 22:19:17 dk-pavilion-laptop gnome-shell[5247]: meta_later_remove: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_get_window_group_for_display: assertion 'compositor' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: JS ERROR: TypeError: global.window_group is null
                                                         destroy@/usr/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js:325:13
                                                         _onDestroy@/usr/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js:453:29
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: Object St.Label (0x558aff229c90), has been already disposed — impossible to access it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs.
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: == Stack trace for context 0x558afd7b77d0 ==
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: #0   7ffcfd253fa0 b   resource:///org/gnome/shell/ui/dash.js:85 (1e37684c4ba0 @ 65)
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: #1   558b00853788 i   /usr/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/docking.js:473 (3f0fa9f426f0 @ 22)
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: #2   7ffcfd2561e0 b   self-hosted:1121 (395fd247eec0 @ 432)
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: clutter_seat_get_pointer: assertion 'CLUTTER_IS_SEAT (seat)' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: clutter_stage_get_device_actor: assertion 'device != NULL' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: clutter_seat_get_pointer: assertion 'CLUTTER_IS_SEAT (seat)' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: clutter_stage_get_device_actor: assertion 'device != NULL' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: meta_later_add: assertion 'display' failed
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: **
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: Clutter:ERROR:../clutter/clutter/clutter-main.c:477:_clutter_context_get_default: assertion failed: (ClutterCntx)
июн 16 22:19:18 dk-pavilion-laptop gnome-shell[5247]: Bail out! Clutter:ERROR:../clutter/clutter/clutter-main.c:477:_clutter_context_get_default: assertion failed: (ClutterCntx)
```

**Comment by vanvugt** [comment]
@dkuteynikov, please try uninstalling `apps-menu@gnome-shell-extensions.gcampax.github.com` and then log in again. If the problem still happens then try uninstalling other extensions until the problem stops happening.

**Comment by vanvugt** [comment]
Looks like the same crash as https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/6786

**Comment by vanvugt** [comment]
The crash is https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/5560 (fixed in gnome-shell 44.0).
